addons/60fps/addon.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "Alt+GreenFlag 60FPS player mode",
-  "description": "Alt+Click the green flag to toggle 60FPS.",
+  "name": "Custom FPS player mode",
+  "description": "Alt+Click the green flag to toggle Custom FPS.",
```

Comment:
I don't think "Custom FPS" should be capitalized.

Reply:
fps was caps before, and custom is first word of name. Player and Mode should maybe be caps

Reply:
I think he was referring to the description.

Reply:
@mxmou were you? i can change that

Reply:
> I think he was referring to the description.

I was.

Reply:
i made the `C` in custom lowercase, but the fps was caps in the notice as well